### ISSUE TITLE
`minetest.disconnect_player` frames reason text as an error message

### ISSUE THREAD

**Issue by Wuzzy2** [minetest/minetest#14986]
### Minetest version

```true
5.9.0
```

### Irrlicht device

X11

### Operating system and version

GNU/Linux

### CPU model

Intel Core i5-2500S CPU 2.7 GHz × 4

### GPU model

AMD Turks (512 MiB)

### Active renderer

OpenGL 4.5

### Summary

If you get disconnected via `minetest.disconnect_player(player, "Some reason text.")`, Minetest disconnects you but the message is framed as an error message. It looks something like this:

"An error has occured: Access denied. Reason: Some reason text.".

I think this is wrong; the message should not be framed as an error but as a generic message. Specifically, the "An error has occured:" must go.

This is because disconnecting a player like this is normal and not done because there's an error or bug in the software.

### Steps to reproduce

1. Install `luacmd`
2. Do the command: `/lua minetest.disconnect_player(me, "This is the disconnect reason text.")`

**Comment by Gamer-Kold** [comment]
I made simple addition to ui.lua to check for `Access denied` in the error message and change the window title accordingly
![image](https://github.com/user-attachments/assets/ddca0bd7-9689-4dc8-9975-856db1f45215)
are there any problems with this approach or am I good to go ahead and make a pull request? Its a pretty tiny change.

**Comment by aabs82** [comment]
I'm a beginner in open-source software development and I'm very interested in this issue. I can give it a try, right?

**Comment by rubenwardy** [comment]
you don't need to ask for permission to try something

**Comment by CrazyladMT** [comment]
EDIT: everything was wrong in this comment lol

**Comment by Wuzzy2** [comment]
No. This bug was not fixed. The text "An error has occured:" still appears as of 155067130. Even the previous screenshot shows that.

Please read the issue text again.

**Comment by CrazyladMT** [comment]
Sorry, my bad. I guess I am blind lol. I could fix it since the other two people seem to not have opened a PR already.

**Comment by CrazyladMT** [comment]
Fixed. I'll open a PR shortly.

<img width="653" height="437" alt="Image" src="https://github.com/user-attachments/assets/50f85a5d-4bcf-4bac-b146-9a1ccf72a8ef" />
